packages/xarc-app-dev/src/lib/dev-admin/dev-server.ts

```diff
@@ -11,6 +12,7 @@ const electrodeServer = optionalRequire("electrode-server");
 const Hapi = optionalRequire("@hapi/hapi");
 const Koa = optionalRequire("koa");
 const express = optionalRequire("express");
+const http = optionalRequire("http");
```

Comment:
`http` is a built-in module, no need to `optionalRequire` it.
also, use `import`.